include/z64save.h

```diff
@@ -645,12 +647,17 @@ typedef enum LinkAge {
 #define EVENTCHKINF_SONGS_FOR_FROGS_STORMS ((EVENTCHKINF_SONGS_FOR_FROGS_INDEX << 4) | EVENTCHKINF_SONGS_FOR_FROGS_STORMS_SHIFT)
 
 // 0xDA-0xDE
-#define EVENTCHKINF_DA_DB_DC_DD_DE_INDEX 13
-#define EVENTCHKINF_DA_MASK (1 << 10)
-#define EVENTCHKINF_DB_MASK (1 << 11)
-#define EVENTCHKINF_DC_MASK (1 << 12)
-#define EVENTCHKINF_DD_MASK (1 << 13)
-#define EVENTCHKINF_DE_MASK (1 << 14)
+#define EVENTCHKINF_SKULLTULA_REWARD_INDEX 13
+#define EVENTCHKINF_SKULLTULA_REWARD_10_SHIFT 10
+#define EVENTCHKINF_SKULLTULA_REWARD_20_SHIFT 11
+#define EVENTCHKINF_SKULLTULA_REWARD_30_SHIFT 12
+#define EVENTCHKINF_SKULLTULA_REWARD_40_SHIFT 13
+#define EVENTCHKINF_SKULLTULA_REWARD_50_SHIFT 14
```

Comment:
I don't like having defines for these

Reply:
Is this a deal-breaker? There are plenty of other shift defines elsewhere in this section.

Reply:
Yep, and I don't particularly like it's usage those cases either.

I feel there's no reason to create defines for these because the values are only ever relevant to exactly one section of code (this file). It just makes the code harder to follow logically for 0 gain.

Reply:
I think it can make understanding the code easier (a comment would also work).